main.tf

```diff
@@ -38,6 +38,11 @@ locals {
   bootstrap_servers = [
     replace(confluentcloud_kafka_cluster.cluster.bootstrap_servers, "SASL_SSL://", "")
   ]
+
+  # Since Mongey's Confluent provider doesn't provide rest endpoint url
+  # we should build it ourselves basing on the assumption that domain
+  # name is the same as in bootstrap_servers
+  rest_api_endpoint = replace(confluentcloud_kafka_cluster.cluster.bootstrap_servers, "/^SASL_SSL:\\/\\/(.*):([0-9]+)$/", "https://$1:443")
```

Comment:
In general, I'm not a fan of generating this inside of a module based upon assumptions.

Reply:
Me too. But I think it is still better than copying this line from one dependent project to another.

Reply:
Do you have an example of where we are using this in other terraform?